attest/attest.go

```diff
@@ -224,11 +224,16 @@ var (
 
 // TPMInfo contains information about the version & interface
 // of an open TPM.
+// FWVersionMajor and FWVersionMinor describe the firmware version of
+// the TPM, but are only available for TPM 2.0 devices.
 type TPMInfo struct {
 	Version      TPMVersion
 	Interface    TPMInterface
 	VendorInfo   string
 	Manufacturer TCGVendorID
+
+	FWVersionMajor int
```

Comment:
nit: Maybe don't abbreviate? These are going to be used sufficiency rare and I don't get FW -> Firmware

```
FirmwareVersionMajor int
FirmwareVersionMinor int
```

Reply:
Done.